### PR TITLE
remove Bloomberg from list of grants as there is no way to apply

### PR DESCRIPTION
**Pull request by matkoniecz**
It is great that Bloomberg supports Open Source but I think listing it here is not helpful.

There is no way to apply and bothering Bloomberg with "please fund also me" email is unlikely to work - if anything, they will decide to do something else with their money

### CODE REVIEW

**Comment by matkoniecz** [comment]
should I make PRs for such removals or just commit directly?

see also #67

**Comment by RichardLitt** [comment]
I've let someone at Bloomberg who I know know. I think PRs help, because they let others know about changes.

**Comment by matkoniecz** [comment]
Oh, and if they are happy to get funding proposals then I will definitely suggest them one (after confirming that they actually use it heavily - but I bet that answer is "yes")

**Comment by matkoniecz** [comment]
> I think PRs help, because they let others know about changes.

what about minor ones like https://github.com/ralphtheninja/open-funding/commit/e4281f577ea08e9492ff81d070b0d221c9da1e3b ? Also put it through PRs?

**Comment by RichardLitt** [comment]
I think anything that might be useful to a watcher would be good to PR. I'm happy to confirm small things, in any event.

**Comment by RichardLitt** [comment]
Confirming for now - can re-add if something changes.

File: README.md [hunk 1]
```diff
@@ -577,49 +571,6 @@ Every chapter interprets "awesome" for itself. As such, awesome projects include
 
 N/A
 
-## B
```
Comment (ralphtheninja) [inline README.md:580]: Maybe we should also remove all the empty sections. This was the only entry for B so the section got removed and there are other empty sections iirc.

**Pull request merged**